sdk/resourcemanager/mobilenetwork/armmobilenetwork/CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # Release History
 
+## 1.0.0 (2022-12-23)
+### Breaking Changes
+
+- Type of `LocalDiagnosticsAccessConfiguration.HTTPSServerCertificate` has been changed from `*KeyVaultCertificate` to `*HTTPSServerCertificate`
+- Const `BillingSKUEdgeSite2GBPS` has been removed
+- Const `BillingSKUEdgeSite3GBPS` has been removed
+- Const `BillingSKUEdgeSite4GBPS` has been removed
+- Const `BillingSKUEvaluationPackage` has been removed
+- Const `BillingSKUFlagshipStarterPackage` has been removed
+- Const `BillingSKULargePackage` has been removed
+- Const `BillingSKUMediumPackage` has been removed
+- Const `PlatformTypeBaseVM` has been removed
+- Function `*MobileNetworksClient.BeginListSimIDs` has been removed
+- Function `*PacketCoreControlPlaneVersionsClient.NewListByResourceGroupPager` has been removed
+- Function `*SimsClient.NewListBySimGroupPager` has been removed
+- Struct `KeyVaultCertificate` has been removed
+- Struct `MobileNetworksClientListSimIDsResponse` has been removed
+- Struct `PacketCoreControlPlaneVersionsClientListByResourceGroupResponse` has been removed
+- Struct `SimIDListResult` has been removed
+- Struct `SimsClientListBySimGroupResponse` has been removed
+- Field `MobileNetwork` of struct `PacketCoreControlPlanePropertiesFormat` has been removed
+- Field `RecommendedVersion` of struct `PacketCoreControlPlaneVersionPropertiesFormat` has been removed
+- Field `VersionState` of struct `PacketCoreControlPlaneVersionPropertiesFormat` has been removed
```

Comment:
Is it possible to update the changelog with latest changelog tool?

Reply:
updated